code/modules/mob/dead/observer/observer.dm

```diff
@@ -206,12 +206,25 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 
 // This is the ghost's follow verb with an argument
 /mob/dead/observer/proc/ManualFollow(atom/movable/target)
-	if(target && target != src)
-		if(orbiting && orbiting == target)
-			return
+	if (!istype(target))
+		return
+
+	var/icon/I = icon(target.icon,target.icon_state,target.dir)
+
+	var/orbitsize = (I.Width()+I.Height())/2
+	orbitsize -= (orbitsize/32)*8

```

Comment:
Ah, I'm used to magicnumber 32s being 32 pixels, as in world.icon_size specifically.
This lets me make a different point though

```/32``` should be ```*0.03125``` :P